scripts/get_hashsets.js

```diff
@@ -1,16 +1,18 @@
 "use strict";
 
-const AppConstants = require("./app-constants").init();
-
 const fs = require("fs");
-const path = require("path");
 const request = require("request");
+const S3 = require("aws-sdk/clients/s3");
 
-const pkg = require("./package.json");
+const AppConstants = require("../app-constants").init();
+const pkg = require("../package.json");
 
 const HIBP_AUTH = `Bearer ${AppConstants.HIBP_API_TOKEN}`;
 const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
-const BREACH_HASHSET_DIR = "breach_hashsets";
+
+const s3 = new S3();
+const BREACH_HASHSET_DIR = "../breach_hashsets";
+const BREACH_HASHSET_BUCKET_NAME = "mozilla.breach_alerts.stage.breach_hashsets";
```

Comment:
Another discussion topic. At what point does it make sense to scope some of these constants to a file vs merge upwards into app-constants.js and .env-dist?
Do we expect others/ops to want to modify the hashset directory location or S3 bucket name?

@nhnt11 Thoughts?

Reply:
I keep constant values in a file until we're re-using them somewhere else. It keeps the "global" constant values less noisy until necessary.